.ci_support/migrations/python312.yaml

```
@@ -0,0 +1,43 @@
+migrator_ts: 1695046563
+__migrator:
+    migration_number: 1
+    operation: key_add
+    primary_key: python
+    ordering:
+        python:
+            - 3.6.* *_cpython
+            - 3.7.* *_cpython
+            - 3.8.* *_cpython
+            - 3.9.* *_cpython
+            - 3.10.* *_cpython
+            - 3.11.* *_cpython
+            - 3.12.* *_cpython  # new entry
+            - 3.6.* *_73_pypy
+            - 3.7.* *_73_pypy
+            - 3.8.* *_73_pypy
+            - 3.9.* *_73_pypy
+    paused: true
+    use_local: True
+    longterm: True
+    pr_limit: 20
+    max_solver_attempts: 3  # this will make the bot retry "not solvable" stuff 12 times
+    exclude:
+      # this shouldn't attempt to modify the python feedstocks
+      - python
+      - pypy3.6
+      - pypy-meta
+      - cross-python
+      - python_abi
+    exclude_pinned_pkgs: false
+    additional_zip_keys:
+      - channel_sources
+
+python:
+  - 3.12.* *_cpython
+# additional entries to add for zip_keys
+numpy:
+  - 1.26
+python_impl:
+  - cpython
+channel_sources:
+  - py312-migration,conda-forge
```

This is going to need to point to the right label
```suggestion
  - conda-forge/label/python_rc,conda-forge
```